_specifications/lsif/0.4.0/specification.md

```diff
@@ -987,7 +987,7 @@ For tools processing the dump and importing it into a database it is sometime us
 For the following example
 
 ```ts
-funciton foo(x: number): void {
+function foo(x: number): void {
```

Comment:
I really can't tell if this is an intentional or accidental typo. If it's intentional, it'd be really helpful if the prose made it explicit.